Controller/BaseController.php

```diff
@@ -128,6 +128,13 @@ protected function getThemeBundle()
      */
     public function createForm($type, $data = null, array $options = array())
     {
+        try {
```

Comment:
I'm assuming the exception caught is some sort of ServiceDoesNotExist.

Rather do something like if (is_string($type) && $this->container->has($type)) which is simpler and cleaner.